jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -618,6 +618,12 @@ private static void ensureProperties(boolean isInitialization) {
 	initializedProperties.put("java.specification.name", "Java Platform API Specification"); //$NON-NLS-1$ //$NON-NLS-2$
 	initializedProperties.put("com.ibm.oti.configuration", "scar"); //$NON-NLS-1$
 
+	/*[IF CRIU_SUPPORT]*/
+	initializedProperties.put("org.eclipse.openj9.criu.isCRIUCapable", "true"); //$NON-NLS-1$ //$NON-NLS-1$
+	/*[ELSE] CRIU_SUPPORT
+	initializedProperties.put("org.eclipse.openj9.criu.isCRIUCapable", "false"); //$NON-NLS-1$ //$NON-NLS-1$
```

Comment:
I would hope any app/test assume `org.eclipse.openj9.criu.isCRIUCapable` is `false` if the system property is not set, then there is no need to set it `false` explicitly.

Reply:
Its not set. Its just that it will return null. If @llxia is fine with that then I can remove the else case.

Reply:
I guess Lan will have to deal with the NULL case anyways, for JDKs that dont have this property at all so Ill just remove it.